permit_intelligence_system.py

```
@@ -0,0 +1,471 @@
+    
+    def _is_investment_relevant(self, permit: Dict) -> bool:
+        """Determine if permit is relevant for investment analysis"""
+        permit_type = permit.get('permit_type', '').upper()
+        permit_subtype = permit.get('permit_sub_type', '').upper()
+        work_desc = permit.get('use_desc', '').upper()
+        valuation = permit.get('valuation', '')
+        
+        # Parse valuation
+        try:
+            val_amount = float(str(valuation).replace('$', '').replace(',', '')) if valuation and valuation != 'N/A' else 0
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)